index.html

```diff
@@ -1,13 +1,76 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Survey Form</title>
   <link rel="stylesheet" href="index.css">
 </head>
+
 <body>
-  
+  <div class="main-content">
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <form action="" id="survey-form">
+
+      <div class="name">
```

Comment:
Use semantic HTML tags. For example, here I would advise you to use the <main></main> tag. These will improve the web accessibility of your app.

Reply:
- [ ] Hi @rosemutai, I still see the same unclear comment. Kindly rephrase it and make it **_polite_** and with **_clear_** meaning passed across. 😄